src/templates/utils/stylingUtils.ts

```diff
@@ -0,0 +1,32 @@
+import classNames from "classnames"
+import _ from "lodash"
+
+const OVERRIDDEN_CLASS_NAMES = [
+  "bp-section",
+  "bp-container",
+  "bp-sec-button",
+  "is-inverted",
+  "row",
+  "col",
+  "sgds-icon",
+]
+const UNSTYLED_CLASS_NAMES = [
+  "has-text-secondary",
+  "bg-secondary",
+  "is-secondary",
+]
+
+interface StylesProps {
+  [key: string]: string
+}
+
+export const getClassNames = (styles: StylesProps, classes: string[]) => {
+  // Generates relevant class, with classes using the provided styles
+  return classNames(
+    classes
+      .filter((className) => !UNSTYLED_CLASS_NAMES.includes(className))
+      .map((className) => styles[className])
+      .concat(_.intersection(classes, OVERRIDDEN_CLASS_NAMES))
+      .concat(_.intersection(classes, UNSTYLED_CLASS_NAMES))
```

Comment:
could you help me understand why we still need this? if we omit hte classname, won't our internal classname take precedence by default?

Reply:
resolved offline - `UNSTYLED_CLASS_NAMES` are css classes which are dynamically injected into our stylesheet at runtime (`siteColorUtils`); we'll add a comment + rename the variable for clarity

Reply:
Renamed in https://github.com/isomerpages/isomercms-frontend/pull/895/commits/c95be74437ce74891ad28a3b66de6d32deaf206b